env-check.sh

```diff
@@ -31,28 +34,55 @@ func_check_python_pkg(){
 
 func_check_file(){
     if [ -e "$1" ]; then
-        return
+        return 0
     fi
-    out_str="$out_str""Optional: Enable dynamic debug logs in \e[31m $1 \e[0m file\n\tFor example,\n\toptions snd_sof dyndbg=+p\n\toptions snd_sof_pci dyndbg=+p\n"
+    case $1 in
+        "$dyn_dbg_conf")
+            out_str="$out_str""\tOptional: Enable dynamic debug logs in \e[31m$1\e[0m file\n"
+            out_str="$out_str""\tFor example:\n\t\toptions snd_sof dyndbg=+p\n\t\toptions snd_sof_pci dyndbg=+p\n"
```

Comment:
I think we can provide a copy of sof-dyndbg.conf in our repo for user reference.
PS: user can put this option in any conf files. So better also provide a simple cmd line here with the files provide.